nemo/lightning/megatron_parallel.py

```diff
@@ -588,8 +588,6 @@ def init_ddp(self):
                     module.config,
                     self.ddp_config,
                     module,
-                    data_parallel_group=parallel_state.get_data_parallel_group(with_context_parallel=True),
-                    expert_data_parallel_group=parallel_state.get_data_modulo_expert_parallel_group(),
```

Comment:
could you describe why these aren't wanted? is there an existing issue that this now fixes?

Reply:
Mcore ddp __init__ at https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/core/distributed/distributed_data_parallel.py#L37 no longer supports these args